model/src/main/java/oracle/kubernetes/weblogic/domain/v2/BaseConfiguration.java

```diff
@@ -71,14 +64,15 @@ void fillInFrom(BaseConfiguration other) {
     if (other == null) return;
 
     if (serverStartState == null) serverStartState = other.getServerStartState();
-    if (overrideStartPolicyFrom(other)) serverStartPolicy = other.getServerStartPolicy();
+    if (overrideStartPolicyFrom(other)) setServerStartPolicy(other.getServerStartPolicy());
```

Comment:
you don't have to fix this now, we can do it post-2.0, but we should not have if statements with no braces

Reply:
Hmmm...  Good natured disagreement, perhaps.  With the enforced formatting, it's hard to see how braces add to readability when there is just one line.